libs/viewer/src/SimpleViewer.cpp

```diff
@@ -1644,7 +1646,17 @@ void SimpleViewer::updateUserInterface() {
     // At this point, all View settings have been modified,
     //  so we can now push them into the Filament View.
     applySettings(mSettings.view, mView);
-    mView->setIblOptions(mSettings.ibl);
+
+    // Set IBL options
+    {        
+        float intensity = mSettings.ibl.iblTintAndIntensity.w;
+        filament::math::float3 linearBgColor = filament::Color::toLinear(mSettings.viewer.backgroundColor);
+        mSettings.ibl.iblTintAndIntensity.r = linearBgColor.r;
+        mSettings.ibl.iblTintAndIntensity.g = linearBgColor.g;
+        mSettings.ibl.iblTintAndIntensity.b = linearBgColor.b;
```

Comment:
Filament math types can be swizzled
```suggestion
        mSettings.ibl.iblTintAndIntensity.rgb = filament::Color::toLinear(mSettings.viewer.backgroundColor);
```

Reply:
It does not compile.

Reply:
`mSettings.ibl.iblTintAndIntensity.rgb = linearBgColor;` works

Reply:
Doh, of course that phrasing produced a 
```
/Users/gaborvalasek/Depot/filament_shapr/libs/viewer/src/SimpleViewer.cpp:1654:19: error: no member named 'ibl' in 'filament::viewer::Settings'
        mSettings.ibl.iblTintAndIntensity.rgb = linearBgColor;
```
since it should read `mSettings.lighting.iblOptions.iblTintAndIntensity.rgb = linearBgColor;` with the current code. Fixed in https://github.com/shapr3d/filament/pull/50/commits/eb2b6c557c21f7be0f28de45d358b8cb001b4c96 .